### ISSUE TITLE
Assigning text's color

### ISSUE THREAD

**Issue by rubenovd** [stevenwanderski/bxslider-3#164]
I was wondering what will be the best approach to texts with custom colors per slide? Integrated into Drupal, we currently have the option to add a title, description and upload an image. Given that not all images are equal I was thinking of somehow pass the colors as hex value or something, or maybe somehow get the IDs per each element and then add those colors in a CSS file? 

Also, I noticed that when I added some text with forced breaks, it showed up as: \r\n   Now my question is, and sorry if make nonsense, will it be possible to pass color as a string as part of the text to bxslider?

Coding is not my strength but so far we manage to get at this point. Thanks in advance for your answers.

**Comment by drKnoxy** [comment]
tl;dr - please close this iss

i'm not head of the project but i can just about guarantee that it'll never pass a color to be applied to text. 
look into using css to accomplish color changes

if you don't want to add classes, or can't try using css3 nth-child selectors.

please close this ticket
